src/SFA.DAS.EAS.Portal.Worker/Properties/launchSettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "profiles": {
+    "SFA.DAS.EAS.Portal.Worker": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "APPSETTING_LoggingRedisConnectionString": "localhost:6379"
```

Comment:
The build is set up to replace this?

Reply:
The Redis connection string is set via environment variables in the environments.